File: baekjoon/bronze/level2/test_1100/Main.java
package bronze.level2.test_1100;

import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int sum = 0;
        char[] arr = new char[8];
        for (int i = 0; i < 8; i++) {
            arr = sc.nextLine().toCharArray();

            if (i % 2 == 0) {
                for (int j = 0; j < 8; j++) {
                    if (j % 2 == 0 && arr[j] == 'F') {
                        sum++;
                    }
                }
            } else {
                for (int j = 0; j < 8; j++) {
                    if (j % 2 == 1 && arr[j] == 'F') {
                        sum++;
                    }
                }
            }
        }

        System.out.print(sum);
    }
}

File: baekjoon/bronze/level2/test_1100/kotlin/Main.kt
package baekjoon.bronze.level2.test_1100.kotlin

fun main() {
    val arr = Array(8) { CharArray(8) }
    var count = 0

    for (i in 0..7) {
        arr[i] = readln().toCharArray()

        for (j in 0..7) {
            if (arr[i][j] != 'F') {
                continue
            }

            if ((i % 2 == 0 && j % 2 == 0) || (i % 2 == 1 && j % 2 == 1)) {
                count++
            }
        }
    }

    print(count)
}
